### DISCUSSION TITLE
Is it possible to make a selection of feature-files that cucumber tests from the commandline?

### DISCUSSION THREAD

**Discussion by exo-cortex** [cucumber-rs/cucumber#330] [Q&A]
I was trying to understand how I can let cucumber "see" only a subset of feature files. Like sending a pattern to cucumber that expands to a certain set of feature-files. Like we can to in the commandline with wildcards etc.
The internal Code of the parser is very complicated to read though. That's why I thought it's better to ask here instead first.

**Comment by tyranron** [comment]
You don't need to look into sources for that. However, if you would like too, here is the piece responsible for that:  https://github.com/cucumber-rs/cucumber/blob/v0.20.2/src/parser/basic.rs#L99-L102

Usually, it's enough either to run the `--help` command or refer to the ["CLI" chapter in the Book](https://cucumber-rs.github.io/cucumber/current/cli.html):
```
  -i, --input <glob>
          Glob pattern to look for feature files with. By default, looks for `*.feature`s in the path configured tests runner
```
